src/datarecord.cpp

```diff
@@ -588,6 +588,24 @@ void datarecord::schedule(reclist& thisi, double maxtime,
   }
 }  
 
+/*  Insert observation records triggered from evtools
+ * 
+ */
+
+void insert_observations(reclist& thisi, mrgsolve::evdata& ev, const size_t start, 
+                         const bool put_ev_first) {
+  const int total = ev.addl + 1;
+  int nextpos = put_ev_first && (ev.evid!= 1 && ev.evid!=4) ? -1000000000 : 1000000000;
```

Comment:
I see that -1000000000 and 1000000000 align with the handling of `addl_ev_first` in `datarecord::schedule`. Okay.